### PR TITLE
Feature/104 fe make the modalidad section in student detail view to be showing real data

### PR DESCRIPTION
**Pull request by FabrizioContu**

### CODE REVIEW

File: src/components/studentDetailCards/modalitySection/modalityCard.tsx [hunk 1]
```diff
@@ -1,12 +1,38 @@
...
+
+    fetchModality()
+  }, [studentUUID])
+
+  return (
+    <div className="flex flex-col gap-3" data-testid="ModalityCard">
+      <h3 className="font-bold text-lg">Modalidad</h3>
+      <div className="flex-col items-center ">
+        {modalityData.modality.map((modality) => (
+          <div key={uuidv4()} className="flex gap-3 py-1">
```
Comment (treee97) [inline src/components/studentDetailCards/modalitySection/modalityCard.tsx:28]: We need to ask to the backend team to review the modality endpoint once again. We should be receiving an array of objects and each object should contain an uuid and the modality name. Besides that everything else is pretty good 👍 😃

**Review by llopFilms — APPROVED** [review]
- uuid és per posar una id a les modalitats i tinguis keys per la llista oi?
- Hauria de venir una id des de backend
- És un tema que s'hauria de solucionar
- Tot ok! 👍🏽

File: src/components/studentDetailCards/modalitySection/modalityCard.tsx (hunk 1 above)
Comment (FabrizioContu) [reply src/components/studentDetailCards/modalitySection/modalityCard.tsx:28]: Thank you, I agree. I'll forward your comment to BackEnd.

**Review by jordimorillo — DISMISSED** [review]
Good job guys

**Comment by FabrizioContu** [comment]
I had to resolve Interface.ts conflict due to the merge of "projects" branch before this one.

**Review by treee97 — APPROVED** [review]

**Review by llopFilms — APPROVED** [review]

**Pull request merged**